JSONModel/JSONModel/JSONModel.m

```diff
@@ -327,7 +328,12 @@ -(BOOL)__importDictionary:(NSDictionary*)dict withKeyMapper:(JSONKeyMapper*)keyM
 
             // check for custom setter, than the model doesn't need to do any guessing
             // how to read the property's value from JSON
-            if ([self __customSetValue:jsonValue forProperty:property]) {
+            NSError* customSetErr = nil;
```

Comment:
Do you use any formatter tool to take care of formatting. Something like uncrustify?